doc/dev/python_scheduling/autotune_pointwise.py

```diff
@@ -89,7 +89,7 @@ def inner_fn():
         if config is not None:
             vectorization_factor, unroll_factor = config
             schedule_params.vectorization_factor = vectorization_factor
-            schedule_params.unroll_factor = unroll_factor
+            schedule_params.unroll_factor_inner = unroll_factor
```

Comment:
do you want to extend to include two unroll paras? `unroll_factor_inner` and `unroll_factor_outer`